scripts/prepare_image.st

```diff
@@ -19,7 +19,7 @@ FileStream fileIn: 'SOHCharacter.cs'.
 (Smalltalk at: #Metacello) new
   baseline: 'BabylonianProgramming';
   repository: 'filetree:///home/runner/work/babylonian-programming-smalltalk/babylonian-programming-smalltalk/packages';
-  load: 'full'.
+  load: 'tests'.
```

Comment:
This is also the wrong version. It should be 'full'

Reply:
how can I fix this? I tried merging dev and liveprog into this branch yesterday.

Reply:
As the merge already happened, you can simply change it back and commit it as a correction :)